Tests/base/NSCharacterSet/general.m

```diff
@@ -88,8 +88,49 @@ int main()
        ![theSet characterIsMember: '#'],
        "Check custom set");
 
-
-  
+  theSet = [NSCharacterSet URLFragmentAllowedCharacterSet];
+  NSString *setString = @"0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ?/:@-._~!$&'()*+,=";
+  NSUInteger i = 0;
+  for(i = 0; i < [setString length]; i++)
+    {
+      char c = [setString characterAtIndex: i];
+      if([theSet characterIsMember: c] == NO) // , [msg cStringUsingEncoding: NSUTF8Encoding]);
+	{
+	  NSLog(@"%c is not in set", c);
```

Comment:
This doesn't look like a real automated test. It should fail on wrong results not just print a message.

Reply:
Fixed